blackbox/docs/src/doc_tests/process_test.py

```diff
@@ -69,6 +69,28 @@ def retry_sql(client, statement):
     raise last_error
 
 
+def check_cluster_size(client, expected_size):
```

Comment:
Maybe rename to `wait_for_cluster_size(client, expected_size, timeout_in_s=20)` to clarify that this method will actually block (or timeout).

Though it would then probably have to raise an exception if the timeout is reached - or the docstring should mention how the return value behaves